app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_split = command.split()
+    if len(command_split) != 3:
```

Comment:
You can simplify this like so:
```
if len(command_breakdown) == 3 and command_breakdown[0] == "mv":
  rest of logic here
OR
if len(command_breakdown) != 3 or command_breakdown[0] != "mv":
   raise ValueError..
rest of logic below

Reply:
isn't it better to separate these errors